frontend/src/util/foodItemRequests.ts

```diff
@@ -0,0 +1,92 @@
+import axios from "axios";
+import {FoodItem} from "../model/FoodItem";
+import moment from "moment/moment";
+import {FoodItemFormData} from "../model/FoodItemFormData";
+
+const API_URL = "/api/food";
+
+export async function fetchAllFoodItems(setAppIsLoading: React.Dispatch<React.SetStateAction<number>>) {
+    let foodItems: FoodItem[] = [];
+
+    setAppIsLoading(oldValue => oldValue++);
+    try {
+        const response = await axios.get(API_URL);
```

Comment:
Possibly integrate catch / finally directly into the axios call

Reply:
Is is the same code as `axios.get... .then .catch .finally` - just written as an `async` function :)